### ISSUE TITLE
Add support for VS Code Insiders in the editor

### ISSUE THREAD

**Issue by SebbeJohansson** [Facepunch/sbox-issues#6278]
### For?

S&Box

### What can't you do?

When I go into preferences to check what IDE to use, VS Code Insiders don't show up on the list.
![image](https://github.com/user-attachments/assets/d80cbfe6-ea73-48ea-99f4-c839f6f190cc)

### How would you like it to work?

I would like VS Code Insiders to show up on the list as a separate option, or to replace the normal Code.

### What have you tried?

I have tried installing the normal version of VS Code and at that point it is still the only option that shows up.

### Additional context

VS Code Insiders is the preview IDE for Code before its released to the masses. I don't have any other Code instance installed my devices.
https://code.visualstudio.com/insiders/

**Comment by bub-bl** [comment]
You need to run VS Code before running S&box editor to unlock this option

**Comment by SebbeJohansson** [comment]
@bub-bl nope. Vs code insiders never show up. if you have gotten this to work, let me know a way to reproduce.

**Comment by Retroeer** [comment]
why not just add an option to open with whatever the System itself prefers? This would solve every niche editor issue

**Comment by SebbeJohansson** [comment]
@Retroeer good idea, if it works fully with the c# solution.
